data_processing/pathology/refined_table/annotation/generate.py

```diff
@@ -0,0 +1,246 @@
+from data_processing.common.Neo4jConnection import Neo4jConnection
+import data_processing.common.constants as const
+from data_processing.pathology.common.build_geojson_from_bitmap import build_geojson_from_bitmap_pandas, concatenate_regional_geojsons_pandas
+from data_processing.pathology.common.utils import get_add_triple_str
+
+from pyspark.sql.functions import udf, lit, col, first, last, desc
+from pyspark.sql.window import Window
+import pandas as pd
+
+import yaml, os
+
+logger = init_logger()
+
+@click.command()
+@click.option('-t', '--template_file', default=None, type=click.Path(exists=True),
+              help="path to yaml template file containing information required for pathology proxy data ingestion. "
+                   "See data_ingestion_template.yaml.template")
+@click.option('-f', '--config_file', default='config.yaml', type=click.Path(exists=True),
+              help="path to config file containing application configuration. See config.yaml.template")
+@click.option('-l', '--label_file', default='data_processing/pathology/config/regional_etl_configuration.yaml',
+              type=click.Path(exists=True),
+              help="path to label configuration file containing application configuration.")
+@click.option('-p', '--process_string', default='geojson',
+              help='comma separated list of processes to run or replay: e.g. geojson OR concat')
+def cli(template_file, config_file, label_file, process_string):
+    """
+    This module generates geojson table for pathology data based on information specified in the template file.
+
+    Example:
+        python -m data_processing.pathology.refined_table.annotation.generate \
+        --template_file {PATH_TO_TEMPLATE_FILE} \
+        --config_file {PATH_TO_CONFIG_FILE} \
+        --label-file {PATH_TO_LABEL_FILE} \
+        --process_string geojson
+
+    """
+    with CodeTimer(logger, 'generate GEOJSON table'):
+        processes = process_string.lower().strip().split(",")
+        logger.info('data template: ' + template_file)
+        logger.info('config_file: ' + config_file)
+        logger.info('processes: ' + str(processes))
+
+        # load configs
+        cfg = ConfigSet(name=const.DATA_CFG, config_file=template_file)
+        cfg = ConfigSet(name=const.APP_CFG,  config_file=config_file)
+
+        current_time_utc = pd.to_datetime('now')
+        current_time = pd.Timestamp(current_time_utc, tz = 'UTC').tz_convert('US/Eastern')
+
+        if 'geojson' in processes:
+            exit_code = create_geojson_table(current_time, label_file)
+            if exit_code != 0:
+                logger.error("GEOJSON table creation had errors. Exiting.")
+                return
+
+        if 'concat' in processes:
+            exit_code = create_concat_geojson_table(current_time, label_file)
+            if exit_code != 0:
+                logger.error("CONCAT-GEOJSON table creation had errors. Exiting.")
+                return
+
```

Comment:
can definitely see this organization easily extending to point annotations in a clean way! this is great work 🎉